tola/views.py

```diff
@@ -201,6 +201,18 @@ def update_user_session(request):
             logger.error(error)
             return HttpResponse(error, status=500)
         return HttpResponse(status=204)
+    if request.is_ajax() and request.method == "GET": 
+        error = None
+        try:
+            query = request.GET["query"]
+            dump = json.dumps({"result": request.session.get(query)})
+            return HttpResponse(dump, content_type='application/json', status=200)
+        except:
+            error = "Error getting session variables (request params {0})".format(query)
+        if error is not None:
+            logger.error(error)
+            return HttpResponse(error, status=500)
+        return HttpResponse(status=204)
```

Comment:
Could you just get rid of the `if error is not None` and handle the logging and error return in the `except` block?  Then you probably wouldn't need the default `return HttpResponse(status=204)`?